### DISCUSSION TITLE
[Feature Request] Quick "stop mouse capture" hotkey

### DISCUSSION THREAD

**Discussion by alpha-se** [ChristopherKlay/StadiaEnhanced#144] [Ideas & Requests]
Hey, not sure if it's easy to grab keypresses before they're processed by stadia, but the Fullscreen mouse is captured using the Pointer Lock API.
For multi-screen users this means that it's not trivial to go to the other screen... Since stadia runs in the browser, Alt+Tab is also not ideal.

A great feature would be to quickly remove the mouse lock with the press of a (configurable?) button.
Doing so is fairly easy:
document.exitPointerLock();

https://developer.mozilla.org/en-US/docs/Web/API/Pointer_Lock_API

**Comment by ChristopherKlay** [comment]
> Hey, not sure if it's easy to grab keypresses before they're processed by stadia

It sadly isn't, which is also why for example the streaming monitor has no key shortcut currently.

Stadia basically has it's own layer for keyboard/controller support, across the entire page, for multiple purposes like input type switching (which is getting used in games to display the correct input type), UI control (especially via the controller) and other things.

It should be doable to add a button in Stadias own menu (similar to windowed mode/streaming monitor) to unlock the cursor, but that's not exactly ideal either.

> Since stadia runs in the browser, Alt+Tab is also not ideal.

You can actually very easily solve this issue. If you install Stadia as a PWA, it has it's own shortcut and window, allowing you to alt-tab into i.e. your browser just fine.
